config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
I was using this other repo because that was the one pointed by @mcastelino and @egernst to me. If the one you mention is doing the job (support for q35), I think this is great and I can use it.